services/nostr-band/nostr-band-followers.tester.js

```diff
@@ -0,0 +1,28 @@
+import Joi from 'joi'
+import { isMetric } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
```

Comment:
This file looks like we've got 4 tests, but as far as I can see, 2 of them are just the same two tests duplicated. They don't seem to be functionally different.

Reply:
done: https://github.com/badges/shields/pull/9870/commits/63b4d698d3edae15399ad32beffed2ec475e2971